### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/capuanob/libnmea-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please note that we have changed tests -> testsuite.

File: fuzz/corpus/segmentaa [hunk 2]
```diff
@@ -0,0 +1 @@
+$GPRMC,170058.89,V,3554.928,N,08002.496,W,9.4,1.57,050521,,E*41
```
Comment (xansec) [inline fuzz/corpus/segmentaa:1]: Is it necessary to commit all of these seed files? Will one or two be sufficient?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec Made the requested changes! I look forward to your feedback

**Comment by xansec** [comment]
Removed git clone.

**Pull request merged**